src/applications/static-pages/simple-forms/21-10210/App.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+const App = ({ formEnabled }) => {
+  if (formEnabled === undefined) {
+    return <va-loading-indicator set-focus message="Loading..." />;
+  }
+
+  if (formEnabled) {
+    return (
+      <>
+        <p>You can apply online or by mail.</p>
```

Comment:
Small content change here:
`You can submit this form online or by mail.` per the [content doc](https://docs.google.com/document/d/1n5rrOrAmdmj9NavOpR8fbjQgFV88iEV4/edit)

Reply:
Fixed.  Thx for catching this!

Reply:
fixed.